src/ocean/core/SmartEnum.d

```diff
@@ -639,11 +629,11 @@ private template ShortestName ( T ... )
 private template DeclareConstants ( T ... )
 {
     static immutable istring DeclareConstants =
-        enum_id ~ " length = " ~ CTFE.toString(T.length) ~ "; " ~
-        enum_id ~ " min = " ~ CTFE.toString(MinValue!(T)) ~ "; " ~
-        enum_id ~ " max = " ~ CTFE.toString(MaxValue!(T)) ~ "; " ~
-        enum_id ~ " min_descr_length = " ~ CTFE.toString(ShortestName!(T)) ~ "; " ~
-        enum_id ~ " max_descr_length = " ~ CTFE.toString(LongestName!(T)) ~ "; ";
+        "enum length = " ~ CTFE.toString(T.length) ~ "; " ~
+        "enum min = " ~ CTFE.toString(MinValue!(T)) ~ "; " ~
+        "enum max = " ~ CTFE.toString(MaxValue!(T)) ~ "; " ~
+        "enum min_descr_length = " ~ CTFE.toString(ShortestName!(T)) ~ "; " ~
+        "enum max_descr_length = " ~ CTFE.toString(LongestName!(T)) ~ "; ";
```

Comment:
This looks unrelated...

Reply:
It is related to a change earlier in this module - `enum_id` was a constant defined based on D version. Now it can be removed and only possible value used directly.